### PR TITLE
Add method to fs to parse plist

### PR DESCRIPTION
**Pull request by imurchie**
Add `parsePlist` method to `fs` object. I've found myself needing to promisify the plist parser again and again, so it needs to be somewhere. It is a file system method, so it seemed to make sense to be here.

### CODE REVIEW

File: lib/fs.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import _fs from 'fs';
 import rimraf from './rimraf';
 import { mkdir } from './util';
-import _md5 from 'MD5';
+import _md5 from 'md5';
```
Comment (jlipps) [inline lib/fs.js]: weird, was this not working before?

File: lib/fs.js [hunk 2]
```diff
@@ -18,6 +20,7 @@ let fs = {
   rimraf,
   mkdir,
   copyFile: B.promisify(_ncp),
+  parsePlist: B.promisify(bplistParser.parseFile)
```
Comment (jlipps) [inline lib/fs.js]: i think @sebv is going to fight you over this. he didn't like it when i added non-node-fs methods to our fs bucket. i kind of disagree still :-)

**Comment by jlipps** [comment]
apart from the larger issue that sebv and i were discussing, this PR LGTM :+1:

File: lib/fs.js (hunk 1 above)
Comment (imurchie) [inline lib/fs.js]: Seems like I broke it before (when updating `md5` to version 2), and didn't notice. On my local machine it doesn't cause a problem, but it does on the Travis build.

**Comment by imurchie** [comment]
Decided to add plist creation, at which time it made sense to move it into its own bucket.

**Comment by sebv** [comment]
Yeah separate plist is better.

**Pull request merged**